### ISSUE TITLE
Barrels aren't mineable and cannot be upgraded.

### ISSUE THREAD

**Issue by Endeavour360** [ThatGravyBoat/Ironchests#62]
Barrels currently aren't mineable. Just need to create a barrels.json and add it to data/ironchests/tags/blocks and modify data/minecraft/tags/blocks/mineable/pickaxe.json to include the new tag. Files with the modifications made attached.

In terms of upgrades, I didn't look into fixing that. But it would be convenient if upgrades worked for barrels.

[barrels.json](https://github.com/ThatGravyBoat/Ironchests/files/13044028/barrels.json)
[pickaxe.json](https://github.com/ThatGravyBoat/Ironchests/files/13044029/pickaxe.json)

**Comment by LuvNHugz** [comment]
Yeah I've noticed this too. Currently I'm just living with it because I don't want random villagers become fishermen from my barrel. Every barrel I place I make copper. but moving it is pretty obnoxious...

**Comment by Djeguer** [comment]
hi, a question, I found where to place the Barrels.json but I can't find where to put the pickaxe.json, where exactly is this file to change?

thank you

**Comment by muggleorwizard** [comment]
Same issue here with not being able to mine barrels.

**Comment by PinkLasagna03** [comment]
Huge problem because of the intentions of the mod, would be nice if you can at least fix this... 

@Endeavour360 Thanks for the data's! You've found a fix for the upgrades already?
